test/socket.test.js

```diff
@@ -1,29 +1,67 @@
 let io = require('socket.io-client');
 let assert = require('assert');
-const { equal } = require('assert');
+
+
+
+
+    
+
 
 function createClient() {
     let client = io("ws://localhost:3000");
     client.emit("_reset");
     return client;
 }
+// when creating client2 don`t reset data
+function createClient2() {
+    let client = io("ws://localhost:3000");
+    return client;
+}
+
+
 
 describe("on connection", () => {
-    it("should connect", (done) => {
+    it("should connect client 1", (done) => {
         client = createClient();
         client.on('connect', () => {
+          
             assert(client.connected);
             client.disconnect();
         });
         client.on('disconnect', () => {
             done();
         });
     })
+    it("should connect client 2", (done) => {
+        client2 = createClient2();
+        client2.on('connect', () => {
+           
+            assert(client2.connected);
+            client2.disconnect();
+        });
+        client2.on('disconnect', () => {
+            done();
+        });
+    })
 
 
 });
+describe("on reset", () => {
+    it("should reset", (done) => {
```

Comment:
tukaj je potrebno testirati ali reset res ustvari effekt ki ga pričakujemo, reset mora pobrisati iz spomina vse, ti pa tukaj preveriš samo response ki bo true, vendar to ni pravilno.

Reply:
Probaj seveda testirati _data če je res počiščen.